velox/functions/prestosql/DecimalFunctions.cpp

```diff
@@ -163,6 +163,61 @@ struct DecimalDivideFunction {
   uint8_t aRescale_;
 };
 
+template <typename R, typename A, typename B>
+inline static R decimalModulus(A a, B b, uint8_t aRescale, uint8_t bRescale) {
```

Comment:
This function is used only once in a one-liner caller. Let's just fold it into the caller.